homeassistant/components/velbus/climate.py

```diff
@@ -79,3 +85,11 @@ async def async_set_preset_mode(self, preset_mode: str) -> None:
         """Set the new preset mode."""
         await self._channel.set_preset(PRESET_MODES[preset_mode])
         self.async_write_ha_state()
+
+    @api_call
+    async def async_set_hvac_mode(self, **kwargs: str) -> None:
+        """Set the new hvac mode."""
+        if (mode := kwargs.get("hvac_mode")) is None:
+            return
+        await self._channel.set_mode(mode)
```

Comment:
Add a map mapping from Home Assistant HVACMode to strings "cool" and "heat" supported by velbus.
Be sure to catch `KeyError`, and translate that to `ServiceValidationError` to address my previous comment about unsupported modes.

Reply:
remove the none state